tests/unit/plugins/service/test_enricher.py

```diff
@@ -127,20 +162,52 @@ def gce_enricher(config, metrics, mock_http_client):
     (exceptions.GCPHTTPError('404 error'), 0, 1, 'GCPHTTPError: 404 error'),
     ([{}] * 5, 4, 5, 'KeyError: \'networkInterfaces\'')])
 @pytest.mark.asyncio
-async def test_handle_message_event_msg_failures(
-        mocker, gevent_msg, mock_async_sleep, gce_enricher, caplog, response,
-        sleep_calls, logs_logged, err_msg):
-    """Raise error while enriching event message."""
+async def test_handle_message_event_msg_additions_failures(
+        mocker, additions_gevent_msg, mock_async_sleep, gce_enricher, caplog,
+        response, sleep_calls, logs_logged, err_msg):
+    """Raise error while enriching event message with additions."""
     gce_enricher._http_client._get_json_mock.side_effect = response
-    gevent_msg.phase = 'enrich'
 
     with pytest.raises(exceptions.GCPGordonError) as e:
-        await gce_enricher.handle_message(gevent_msg)
+        await gce_enricher.handle_message(additions_gevent_msg)
 
-    assert 'enrich' == gevent_msg.phase
```

Comment:
I removed this assertion as it didn't seem to really test anything? Since the test manually set the `gevent_msg.phase` value via `gevent_msg.phase = 'enrich'` on line 135.